pkg/kor/networkpolicies.go

```diff
@@ -7,14 +7,14 @@ import (
 	"fmt"
 	"os"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
As I see in the rest of the code base, we currently use `metav1` in `pkg/kor/<resource>.go` and `v1` in `pkg/kor/<resource>_test.go`, not sure what was the root reason for this seperation.

So please revert the change in `networkpolicies.go` back to `metav1` and in a future refactor we'll create a convention and adjust all occurences of both `v1` & `metav1`.

Reply:
Ah yes! Thanks, I didn't mean to commit this change. But seems like using `metav1` or `netv1` would be more descriptive. Just my opinion :D